src/test/java/address/testutil/ContextMenuChoice.java

```diff
@@ -0,0 +1,8 @@
+package address.testutil;
+
+/**
+ *
+ */
+public enum ContextMenuChoice {
+        EDIT, TAG, DELETE, CANCEL;
+}
```

Comment:
That's not going to be scalable though. A context menu can grow to have many options.